### PR TITLE
Add sa.hypershift-workload-sa.hive.token.txt

### PR DESCRIPTION
**Pull request by bear-redhat**
Add the missing `sa.hypershift-workload-sa.hive.token.txt` to secret-bootstraper/generator

Ref: https://github.com/openshift/release/pull/33635

/cc hongkailiu

### CODE REVIEW

**Review by hongkailiu — APPROVED** [review]

**Review by hongkailiu — COMMENTED** [review]
/hold

File: core-services/ci-secret-bootstrap/_config.yaml
Comment (hongkailiu) [inline core-services/ci-secret-bootstrap/_config.yaml:2106]:
```suggestion
    sa.hypershift-workload-sa.hive.token.txt:
```

File: core-services/ci-secret-generator/_config.yaml
Comment (hongkailiu) [inline core-services/ci-secret-generator/_config.yaml:162]: Is the service account named `hypershift-workload-sa`?
That sounds odd.
The naming convention is something like `deck` for the component deck, instead of `deck-sa`.
Could we name it `hypershift-workload`, or something like it?

Comment (bear-redhat) [reply core-services/ci-secret-generator/_config.yaml:162]: we can fix it

File: clusters/hive/hypershift.yaml
Comment (hongkailiu) [inline clusters/hive/hypershift.yaml:5]:
```suggestion
  # Token rotation only support ServiceAccount in ci namepsace
```
```suggestion
  # Secret rotation only support ServiceAccount in ci namepsace
```

**Review by hongkailiu — COMMENTED** [review]
You need to delete the old SA again after merge.
I know you have deleted it once in our meeting but it might come back because of git-ops.

**Review by hongkailiu — APPROVED** [review]

**Pull request merged**
